Jenkinsfile

```diff
@@ -20,15 +22,26 @@ pipeline {
         echo 'Checking out project...'
         checkout scm
         echo 'Building...'
-        withMaven(jdk: 'OpenJDK11', maven: 'Maven3.6', mavenSettingsConfig: 'hcomdata-artifactory-maven-settings') {
-          sh 'mvn clean deploy jacoco:report checkstyle:checkstyle'
+        withMaven(jdk: 'OpenJDK11', maven: 'Maven3.6', mavenSettingsConfig: 'hcomdata-oss-settings') {
```

Comment:
Is `hcomdata-oss-settings` something already created in Jenkins?

And should we make one for eg-oss?

Reply:
We have a fresh project there, so its empty, but I think you're right we'll need to use different settings for eg open source. Good spot, thanks